### PR TITLE
10-user-persistence

### LINKED ISSUE #10: 10 - User Persistence
```
As a user
So that I can log back in
I want my user details to be saved
```

### PR DESCRIPTION
**Pull request by alextwilson**
Closes #10

User data is persisted

- Moved sign up tests to own spec
- Created user class
- Removed password confirmation

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class User
+  include DataMapper::Resource
+
+  property :id, Serial
+  property :name, Text
+  property :username, Text, required: true, unique: true
```
Comment (Leigan0) [inline app/models/user.rb:6]: If you are using required at this stage, may be worth doing this with password

**Review by lewmoore — COMMENTED** [review]
:shipit:

**Review by ghost — COMMENTED** [review]
Tick

**Comment by Leigan0** [comment]
💅

**Comment by Gleoman** [comment]
:shipit:

**Pull request merged**